src/3d/qgstessellator.h

```diff
@@ -58,6 +58,7 @@ class _3D_EXPORT QgsTessellator
     double mOriginX, mOriginY;
     bool mAddNormals;
     bool mInvertNormals;
+    bool mAddBackFaces;
```

Comment:
= false

Reply:
This is something I am always wondering about - if we always set such variable in the constructor (possibly with default value), then it seems to me like extra unnecessary effort to add some value also in the class definition?

Reply:
To me it's about future proofing code. It's just another level of safety if someone comes by and adds a secondary constructor in future.

Reply:
(plus, always initialising members means it's easier to spot real issues cause by uninitialised members)